integration_tests/src/main/python/aqe_test.py

```diff
@@ -24,7 +24,7 @@
 # allow non gpu when time zone is non-UTC because of https://github.com/NVIDIA/spark-rapids/issues/9653'
 not_utc_aqe_allow=['ShuffleExchangeExec', 'HashAggregateExec'] if is_not_utc() else []
 
-_adaptive_conf = { "spark.sql.adaptive.enabled": "true" }
+_adaptive_conf = { "spark.sql.adaptive.enabled": "true"}
```

Comment:
Nit: Looks like an inadvertent change
```suggestion
_adaptive_conf = { "spark.sql.adaptive.enabled": "true" }
```